components/org.wso2.carbon.identity.application.authenticator.organization.login/src/main/java/org/wso2/carbon/identity/application/authenticator/organization/login/OrganizationAuthenticator.java

```diff
@@ -291,6 +317,46 @@ public AuthenticatorFlowStatus process(HttpServletRequest request, HttpServletRe
         return super.process(request, response, context);
     }
 
+    private String getIdTokenHint(AuthenticationContext context) {
+
+        if (context.getStateInfo() instanceof OIDCStateInfo) {
+            return ((OIDCStateInfo) context.getStateInfo()).getIdTokenHint();
+        }
+        return null;
+    }
+
+    private String getIdTokenClaim(String idToken, String claimName) {
+
+        String base64Body = idToken.split("\\.")[1];
+        byte[] decoded = Base64.getDecoder().decode(base64Body);
+        Set<Map.Entry<String, Object>> jwtAttributeSet = new HashSet<>();
+        try {
+            jwtAttributeSet = JSONObjectUtils.parse(new String(decoded, StandardCharsets.UTF_8)).entrySet();
+        } catch (ParseException e) {
+            log.error("Error occurred while parsing JWT provided by federated IDP: ", e);
```

Comment:
```suggestion
            log.error("Error occurred while parsing JWT: ", e);
```
since this is a generic method

Reply:
Fixed